windows/renderer/index.html

```
@@ -0,0 +1,116 @@
+            <div class="inspector-title" id="inspectorTitle">Notifications</div>
+            <div class="inspector-subtitle" id="inspectorSubtitle">Attention across workspaces</div>
+          </div>
+          <button class="icon-button" id="closeInspectorButton" title="Close">×</button>
+        </div>
+        <div class="inspector-body" id="inspectorBody"></div>
+      </aside>
+    </div>
+
+    <div class="palette" id="palette" aria-hidden="true">
+      <div class="palette-dialog">
+        <input id="paletteInput" class="palette-input" autocomplete="off" spellcheck="false" placeholder="Command">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)